### DISCUSSION TITLE
[perfromance][contact] Optimize contacts subscriptions

### DISCUSSION THREAD

**Discussion by flexsurfer** [status-im/status-legacy#16617] [General]
Currently, we have lots of subscriptions for contacts with computations, we should move all that into events , we should have only subscriptions where we get contacts from app-db

Related issue: https://github.com/status-im/status-react/issues/10287

![image](https://user-images.githubusercontent.com/11790366/79864138-3b0bea80-83d9-11ea-9674-5726369906b3.png)

**Comment by churik** [comment]
@flexsurfer 
is the issue still relevant according to the new design implementation?

**Comment by flexsurfer** [comment]
yes

**Comment by cammellos** [comment]
Moving to discussion, and we can re-open when ready to be worked on
